docs/rules_cn.md

```diff
@@ -746,6 +746,20 @@
 - 分类: `TIS`, `BUGFIX`
 
 
+## 启用旧版NC更新器（simpleNeighborUpdaterEnabled）
+
+启用旧版NC更新器
+
+在1.19中，使用新版NC更新器意味着不能使用传统的更新抑制器（衷心希望新的原版更新抑制器能研发出来）
```

Comment:
禁掺私货

Reply:
以及通过堆叠大力突破 1000000 连锁更新上限的 1.19 “更新抑制器”是存在的

Reply:
Yes, it is a workable but not practical solution to update suppression, for that new NC updater has changed to BFS(Breadth-First Search), which means that where you operate will be updated firstly, and the suppressed update is very far away from you, item shadowing may be possible, but portal curving, I cannot come up a idea because you have to dig the obsidian by yourself (or by hitting a Wither? but it will be updated in next tick).

By the way, I test `max-chained-neighbor-updates=1` in server, and here is a record

https://user-images.githubusercontent.com/38467722/172602114-23993284-3831-4d62-9334-ff71004bed43.mp4

Reply:
afaik, the new updater is DFS based, and due to lack of exception throwing item shadowing is not possible